docs/conf/australia/2024/news/announcing-speakers-schedule.rst

```diff
@@ -0,0 +1,100 @@
+:template: {{year}}/generic.html
+
+.. post:: September 22, 2024
+   :tags: {{shortcode}}-{{year}}, speakers, tickets, visiting, sponsors
+
+Announcing conference speakers, talks and the full schedule
+============================================================
+
+G'day, documentarians! Today we're excited to announce that we've finalised our speakers for the next Australia conference.
+
+Thank you so much to everyone who took the time to send us a proposal – we literally wouldn't have an event without you!
+
+Our Australian conference is happening in just over 2 months, and we're excited to see everyone again in {{city}}.
+
+Full speaker line-up
+--------------------
+
+Each year we bring a wide range of voices to the Write the Docs. Because the role of "documentarian" looks so different to each of us, we aim for a line-up that spans a good mix of practical, philosophical, and technical topics – with the odd whimsical one thrown in for good measure.
+
+We're really excited about the `presentations <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/speakers/>`_ we've got this year:
+
+.. datatemplate::
```

Comment:
It's gonna make life a lot easier if you add the invited speakers to Pretalx, and we extract the data from there in the same way.

Reply:
Yep, will do once I get their bios and talk abstracts